finality-aleph/src/justification.rs

```diff
@@ -179,14 +182,13 @@ where
 
         let mut notification_stream = futures::stream::select(import_stream, authority_stream);
 
+        let (session_period, _, _) = self.chain_cadence;
```

Comment:
Maybe it's better for chain_cadence to be a struct with 3 named fields instead of a 3-tuple? :)

Reply:
I'm also wondering why is `millisecs_per_block` in `ChainCadence`. I haven't seen it used anywhere.

Reply:
removed